pkg/security/podsecuritypolicy/v1beta1/factory.go

```diff
@@ -110,88 +110,88 @@ func (f *simpleStrategyFactory) CreateStrategies(psp *policy.PodSecurityPolicy,
 }
 
 // createUserStrategy creates a new user strategy.
-func createUserStrategy(opts *policy.RunAsUserStrategyOptions) (user.RunAsUserStrategy, error) {
+func createUserStrategy(opts *policyv1beta1.RunAsUserStrategyOptions) (user.RunAsUserStrategy, error) {
```

Comment:
I'd prefer leaving the import named `policy`, and pointing it to `policy/v1beta1`, so that graduating this code to point to `policy/v1` would be as simple as changing the import (assuming no other changes between v1beta1 and v1).

Reply:
that comment applies throughout. referring to core/v1 as 'corev1' is fine, since that package is already at GA

Reply:
done